composer.json

```diff
@@ -102,6 +106,7 @@
         "stripe/stripe-php": "~2.0|~3.0 If you want to use Stripe install stripe/stripe-php:~2.0|~3.0 library",
         "ext-soap": "* If you want to use Payex extension you have to install soap extension"
     },
+    "minimum-stability": "beta",
```

Comment:
Puli is not stable yet. Either this or require 4 extra packages that are in beta.